monitoring-support/src/main/java/org/terracotta/monitoring/PlatformServer.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Terracotta Core.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.monitoring;
+
+import com.tc.classloader.CommonComponent;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+/**
+ *
+ */
+@CommonComponent
+public class PlatformServer {
+  private final String serverName;
+  private final String hostName;
+  private String hostAddress;
+  private final String bindAddress;
+  private final int bindPort;
+  private final int groupPort;
+  private final String version;
+  private final String build;
+  private final long startTime;
+
+  public PlatformServer(String serverName, String host, String bindAddress, int bindPort, int groupPort, String version, String build, long startTime) {
+    this.serverName = serverName;
+    this.hostName = host;
+    try {
+      this.hostAddress = InetAddress.getByName(host).getHostAddress();
```

Comment:
@jd0-sag explained me that the voltron tree will contain all servers within a stripe. So I guess that if I am on the active, PlatformServer objects from passive are not created from this constructor ?